docs/content/v2.0/admin/yb-admin.md

```diff
@@ -534,18 +534,23 @@ yb-admin -master_addresses <master-addresses> modify_placement_info <placement_i
 ```
 
 - *master-addresses*: Comma-separated list of YB-Master hosts and ports. Default value is `localhost:7100`.
-- *placement_info*: Comma-delimited list of placements for *cloud*.*region*.*zone*. Default value is `cloud1.datacenter1.rack1`.
-- *replication_factor*: The number of replicas for each tablet.
+- *placement_info*: Comma-delimited list of placements for *cloud*.*region*.*zone*. Default value is `cloud1.datacenter1.rack1`. It is not recommended to repeat placement multiple times but instead specify the total count after the colon. However, in the event that the user specifies a placement multiple times, the total count from all mentions is taken.
+- *replication_factor*: The number of replicas for each tablet. This value should be more than the total of replica counts specified in *placement_info*.
 - *placement_id*: The identifier of the primary cluster, which can be any unique string. If not set, a randomly-generated ID will be used.
 
 **Example**
 
 ```sh
 $ ./bin/yb-admin --master_addresses $MASTER_RPC_ADDRS \
     modify_placement_info  \
-    aws.us-west.us-west-2a,aws.us-west.us-west-2b,aws.us-west.us-west-2c 3
+    aws.us-west.us-west-2a:2,aws.us-west.us-west-2b:3,aws.us-west.us-west-2c 8
```

Comment:
We generally add an RF value that is odd. Change `aws.us-west.us-west-2b:3` to `aws.us-west.us-west-2b:2` and the RF to 5.